src/components/EventContainer.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+// Material
+import { withStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+
+import TimeInput from 'material-ui-time-picker'
+
+const styles = {
+  card: {
+    width: 650,
+  },
+  media: {
+    height: 130,
+    width: 650,
+    backgroundColor: '#3f51b5'
+  },
+  content: {
+    height: '380px',
+    width: '100%',
+  },
+  form: {
+    display: 'flex',
+    justifyContent: 'center',
+    flexWrap: 'wrap',
+    alignItems: 'center',
+    flexFlow: 'column'
+  },
+  textField: {
+    marginLeft: '16px',
+    marginRight: '16px',
+    width: 200,
+  },
+  actions: {
+    display: 'flex',
+    justifyContent: 'space-between'
+  }
+};
+
+
+class EventContainer extends React.Component {
+  state = {
```

Comment:
React docs recommend to initialize state in constructor

Reply:
Ok.